tests/Unit/PointwiseFunctions/GeneralRelativity/Test_Ricci.cpp

```diff
@@ -0,0 +1,134 @@
+  CHECK(ricci.get(2, 2) == approx(109.));
+
+  check_tensor_doubles_equals_tensor_datavectors(
+      gr::ricci_tensor(
+          make_spatial_christoffel_second_kind<spatial_dim>(used_for_size),
+          make_deriv_spatial_christoffel_second_kind<spatial_dim>(
+              used_for_size)),
+      ricci);
+}
+
+void test_1d_spacetime_ricci(const DataVector& used_for_size) {
+  const size_t spatial_dim = 1;
+  const auto ricci = gr::ricci_tensor(
+      make_spacetime_christoffel_second_kind<spatial_dim>(0.),
+      make_deriv_spacetime_christoffel_second_kind<spatial_dim>(0.));
+  CHECK(ricci.get(0, 0) == approx(-46.));
+  CHECK(ricci.get(0, 1) == approx(28.));
+  CHECK(ricci.get(1, 1) == approx(-16.));
+
+  check_tensor_doubles_equals_tensor_datavectors(
+      gr::ricci_tensor(
+          make_spacetime_christoffel_second_kind<spatial_dim>(used_for_size),
+          make_deriv_spacetime_christoffel_second_kind<spatial_dim>(
+              used_for_size)),
+      ricci);
+}
+
+void test_2d_spacetime_ricci(const DataVector& used_for_size) {
+  const size_t spatial_dim = 2;
+  const auto ricci = gr::ricci_tensor(
+      make_spacetime_christoffel_second_kind<spatial_dim>(0.),
+      make_deriv_spacetime_christoffel_second_kind<spatial_dim>(0.));
+  CHECK(ricci.get(0, 0) == approx(-406.));
+  CHECK(ricci.get(0, 1) == approx(-32.));
+  CHECK(ricci.get(0, 2) == approx(217.));
+  CHECK(ricci.get(1, 1) == approx(-178.));
+  CHECK(ricci.get(1, 2) == approx(143.5));
+  CHECK(ricci.get(2, 2) == approx(-201.));
+
+  check_tensor_doubles_equals_tensor_datavectors(
+      gr::ricci_tensor(
+          make_spacetime_christoffel_second_kind<spatial_dim>(used_for_size),
+          make_deriv_spacetime_christoffel_second_kind<spatial_dim>(
+              used_for_size)),
+      ricci);
+}
+
+void test_3d_spacetime_ricci(const DataVector& used_for_size) {
+  const size_t spatial_dim = 3;
+  const auto ricci = gr::ricci_tensor(
+      make_spacetime_christoffel_second_kind<spatial_dim>(0.),
+      make_deriv_spacetime_christoffel_second_kind<spatial_dim>(0.));
+  CHECK(ricci.get(0, 0) == approx(-1928.));
+  CHECK(ricci.get(0, 1) == approx(-700.));
+  CHECK(ricci.get(0, 2) == approx(283.));
+  CHECK(ricci.get(0, 3) == approx(940.));
+  CHECK(ricci.get(1, 1) == approx(-1300.));
+  CHECK(ricci.get(1, 2) == approx(82.5));
+  CHECK(ricci.get(1, 3) == approx(968.));
+  CHECK(ricci.get(2, 2) == approx(-629.));
+  CHECK(ricci.get(2, 3) == approx(335.5));
+  CHECK(ricci.get(3, 3) == approx(-1176.));
+
+  check_tensor_doubles_equals_tensor_datavectors(
+      gr::ricci_tensor(
+          make_spacetime_christoffel_second_kind<spatial_dim>(used_for_size),
+          make_deriv_spacetime_christoffel_second_kind<spatial_dim>(
+              used_for_size)),
+      ricci);
+}
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.Ricci.",
+                  "[PointwiseFunctions][Unit]") {
+  const DataVector dv(2);
```

Comment:
Use size 5

Reply:
Do you mean dv(2) -> dv(5)? Either way, what is the purpose of this change?

Reply:
Yes, 2 is still too "special". We've had issues with not testing large enough DataVectors allowing bugs to slip by us

Reply:
This should be changed in existing tests then. I'll make an issue.